vhdbuilder/packer/windows/configure-windows-vhd.ps1

```diff
@@ -1006,6 +1037,7 @@ try
             Get-ToolsToVHD
             Get-PrivatePackagesToCacheOnVHD
             Log-ReofferUpdate
+            Install-WindowsCilium
```

Comment:
Can you put this before the Log-ReofferUpdate? We do that just before each reboot as it tells us ... stuff.

Reply:
Sure.